<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>348</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>601</width>
    <height>348</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>601</width>
    <height>348</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>221</width>
     <height>328</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>109</number>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <row>
    <property name="text">
     <string/>
    </property>
   </row>
   <row>
    <property name="text">
     <string/>
    </property>
   </row>
   <row>
    <property name="text">
     <string/>
    </property>
   </row>
   <row>
    <property name="text">
     <string/>
    </property>
   </row>
   <row>
    <property name="text">
     <string/>
    </property>
   </row>
   <row>
    <property name="text">
     <string/>
    </property>
   </row>
   <row>
    <property name="text">
     <string/>
    </property>
   </row>
   <row>
    <property name="text">
     <string/>
    </property>
   </row>
   <row>
    <property name="text">
     <string/>
    </property>
   </row>
   <row>
    <property name="text">
     <string/>
    </property>
   </row>
   <column>
    <property name="text">
     <string>K(i)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Y(i)</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="btn_random_number">
   <property name="geometry">
    <rect>
     <x>239</x>
     <y>234</y>
     <width>221</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Заполнить случайными числами</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_solve">
   <property name="geometry">
    <rect>
     <x>239</x>
     <y>288</y>
     <width>221</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Расчёт</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_clear">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>234</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Очистить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_exit">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>288</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_img">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>30</y>
     <width>351</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_error">
   <property name="geometry">
    <rect>
     <x>286</x>
     <y>140</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
